### DISCUSSION TITLE
入群问题太难了，被封了。

### DISCUSSION THREAD

**Discussion by ghost** [XTLS/Xray-core#2557] [General]
谁给我解封一下啊？没入群的时候还能看一下，现在看都看不了了。

**Comment by FranzKafkaYu** [comment]
> 谁给我解封一下啊？没入群的时候还能看一下，现在看都看不了了。

ID多少呢，我记得你在群里的呀

**Reply by ghost** [reply]
yiguoapp

**Comment by FranzKafkaYu** [comment]
> yiguoapp  

已经拉进群了~

**Comment by RPRX** [comment]
哈哈哈哈哈，~~风扇干的好事~~，不过人不在多而在精

**Reply by 50infivedays** [reply]
我也被封了 可以拉下我吗

**Reply by ghost** [reply]
@RPRX @yuhan6665 , 由于个人原因， ~~给其他框架提交 PR 时不小心泄漏了信息~~ ，得换个新账号 @yiguous , 麻烦给拉一下。

**Comment by Fangliding** [comment]
本来入群失败是禁一个小时的
~~被xq喵改成12h了（）~~
